iteration/iteration_test.go

```diff
@@ -157,7 +154,7 @@ func (s *TestIterationRepository) TestRootIteration() {
 		SpaceID:    space.ID,
 		StartAt:    &start,
 		EndAt:      &end,
-		UserActive: &userActive,
+		UserActive: false,
```

Comment:
Remove this line, since `UserActive` will automatically be `false`.

Reply:
Removed it here b92020c